components/Header/Header.tsx

```diff
@@ -107,7 +107,7 @@ const Header: React.FunctionComponent<HeaderProps> = ({
           </View>
         )}
 
-        {dimensions.orientation === 'landscape' && (
+        {(
           <View style={{ width: '100%', height: 1, backgroundColor: colors.primary }} />
         )}
```

Comment:
you have to remove those 3 lines, because in `portrait` mode you don't need this horizontal line.

Reply:
oh oops! i didn't actually see that line, but yeah i shouldnt have added it. double extra fixed this time